hazelcast/src/main/java/com/hazelcast/util/LinkedMapBuilder.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.util;
+
+import java.util.Map;
+import java.util.LinkedHashMap;
+
+/***
+ * This class provides builder pattern to create LinkedHashMap
+ *
+ * @param <K> - type of map of the key
+ * @param <V> - type of map of the value
+ */
+public final class LinkedMapBuilder<K, V> {
```

Comment:
Is this class used in Hazelcast or only in JET? In this PR I don't see any usage of LinkedMapBuilder.